pages/index.js

```diff
@@ -1,3 +1,67 @@
-export default function HomePage() {
-  return <h1>Hello from Next.js</h1>;
+import { useState } from "react";
+import styled from "styled-components";
+
+export default function Landingpage() {
+  const [entry, setEntry] = useState("");
+  const [message, setMessage] = useState("");
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    setMessage(entry);
+    setEntry("");
+  };
+  return (
+    <>
+      <container>
+        <Title>Your Travel Journal</Title>
+        <Form
+          onSubmit={handleSubmit}
+          aria-labelledby="Where is your journey going?"
+        >
+          <Label htmlFor="text">City or Country?</Label>
+          <input
+            type="text"
+            required
+            id="text"
+            name="entry"
+            value={entry}
+            placeholder="enter your destination..."
+            onChange={(event) => setEntry(event.target.value)}
```

Comment:
Do you need to control your input? I can also use the "formData" approach in your handleSubmit instead of using the entry state.

Reply:
Hey Felix!
I dont need to control the input for now. I just want that the entry is displayed and that was the first way i thought of implementing it. Can I leave it like this? Or is the "formData" approach recommended?

Reply:
Then I would reccommend the formData approach, then you don't have to juggle around with the two states. You could also just use "event.target.elements.<name of input>.value"